lib/GtkPatch/AboutDialog.vala

```diff
@@ -360,53 +372,51 @@ public class Granite.GtkPatch.AboutDialog : Gtk.Dialog {
     private void update_license () {
         switch (license_type) {
         case Gtk.License.GPL_2_0:
-            set_generic_license("http://www.gnu.org/licenses/old-licenses/gpl-2.0.html", "GPL 2.0");
+            set_generic_license ("http://www.gnu.org/licenses/old-licenses/gpl-2.0.html", "GPL 2.0");
             break;
         case Gtk.License.GPL_3_0:
-            set_generic_license("http://www.gnu.org/licenses/gpl.html", "GPL");
+            set_generic_license ("http://www.gnu.org/licenses/gpl.html", "GPL");
             break;
         case Gtk.License.LGPL_2_1:
-            set_generic_license("http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html", "LGPL 2.1");
+            set_generic_license ("http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html", "LGPL 2.1");
             break;
         case Gtk.License.LGPL_3_0:
-            set_generic_license("http://www.gnu.org/licenses/lgpl.html", "LGPL");
+            set_generic_license ("http://www.gnu.org/licenses/lgpl.html", "LGPL");
             break;
         case Gtk.License.BSD:
-            set_generic_license("http://opensource.org/licenses/bsd-license.php", "BSD");
+            set_generic_license ("http://opensource.org/licenses/bsd-license.php", "BSD");
             break;
         case Gtk.License.MIT_X11:
-            set_generic_license("http://opensource.org/licenses/mit-license.php", "MIT");
+            set_generic_license ("http://opensource.org/licenses/mit-license.php", "MIT");
             break;
         case Gtk.License.ARTISTIC:
-            set_generic_license("http://opensource.org/licenses/artistic-license-2.0.php", "Artistic");
+            set_generic_license ("http://opensource.org/licenses/artistic-license-2.0.php", "Artistic");
             break;
         default:
             if (license != null && license != "") {
-                license_label.set_markup(license + "\n");
-                license_label.show();
+                license_label.set_markup (license + "\n");
+                license_label.show ();
             }
             else
-                license_label.hide();
+                license_label.hide ();
             break;
         }
     }
 
     private void set_generic_license (string url, string license_type) {
-        license_label.set_markup("<span size=\"small\">" + _("This program is published under the terms of the %s license, it comes with ABSOLUTELY NO WARRANTY; for details, visit %s").printf (license_type, "<a href=\"" + url + "\">" + url + "</a></span>\n"));
-        license_label.show();
+        license_label.set_markup ("<span size=\"small\">" + _("This program is published under the terms of the %s license, it comes with ABSOLUTELY NO WARRANTY; for details, visit %s").printf (license_type, "<a href=\"" + url + "\">" + url + "</a></span>\n"));
```

Comment:
I'm not sure how to split this to a newline. I tried something like this, but it's still too long:
```
license_label.set_markup (
            "<span size=\"small\">"
            + _("This program is published under the terms of the %s license, it comes with ABSOLUTELY NO WARRANTY; for details, visit %s")
                .printf (license_type, "<a href=\"" + url + "\">" + url + "</a></span>\n")
        );
```

Any advice on a better way to split this one to follow the 120 rule?

Reply:
Granite.AboutDialog is deprecated and will be removed later so I wouldn't sweat it. You can just leave it :)

Reply:
:+1: added an ignore line for this one, and left it how it is